moz_telemetry/io_modules/decoders/moz_telemetry/ping.lua

```diff
@@ -323,26 +337,35 @@ local function remove_objects(msg, doc, section, objects)
 end
 
 
+local function validate_schema(hsr, msg, doc, version)
```

Comment:
The vacuous schema is no different than any other (whatever file is provided by the schema service is used).  The only difference is if no schema match is found it is used by default.  Would you prefer the schema be baked into this module?